src/app/content/studyGuides/hooks/loadMore.ts

```diff
@@ -60,7 +65,7 @@ export const hookBody: ActionHookBody<
   services.dispatch(actions.receiveSummaryStudyGuides(formattedHighlights, {pagination, filters}));
 };
 
+export const locationChangeHook = actionHook(locationChange, hookBody);
 export const loadMoreHook = actionHook(actions.loadMoreStudyGuides, hookBody);
 export const setSummaryFiltersHook = actionHook(actions.setSummaryFilters, hookBody);
-export const updateSummaryFiltersHook = actionHook(actions.updateSummaryFilters, hookBody);
-export const setDefaultSummaryFiltersHook = actionHook(actions.setDefaultSummaryFilters, hookBody);
+export const receiveStudyGuidesTotalCountsHook = actionHook(actions.receiveStudyGuidesTotalCounts, hookBody);
```

Comment:
@TomWoodward I removed the loadMore action from studyguides/hooks/locationChange and instead hook loadMore into this action which solved the issue I had with populating study guides on reload

Reply:
i thought loadMore was triggered by the dialog itself when it opened? was the issue that it was opening before `receiveStudyGuidesTotalCounts` comes in? that is a problem because that response may indicate that there are no study guides for this book. maybe the modal open indicator needs to include that the counts exist.

separately, just make sure loadmore is correctly called when changing filters and stuff

Reply:
Yes, loadMore is triggered with the dialog is opened manually after page load. The problem is loading the page with the modal in the query—the `studyGuidesOpen` selector evaluates as false because the book/page has not loaded.

Reply:
thats good right?

Reply:
at first yes but once the book/page loads shouldn't it update?